components/EditForm/index.js

```diff
@@ -89,10 +122,29 @@ export default function EditForm({ project, setEditMode }) {
         {errors.title && (
           <StyledErrorMessage>{errors.title.message}</StyledErrorMessage>
         )}
-        <StyledLabel>
-          Image:
-          <StyledInput {...register("image")} defaultValue={project.image} />
-        </StyledLabel>
+
+        {/* <StyledLabel>
+          Cover
+          <StyledInput
+            name="cover"
+            type="file"
+            {...register("cover", { required: !project.cover })}
+          />
+          {project.cover ? (
+            <Image
+              src={project.cover.url}
+              width={project.cover.width}
+              height={project.cover.height}
+              alt={project.title}
+            />
+          ) : (
+            "No picture selected"
+          )}
+          {errors.cover && (
+            <StyledErrorMessage>{errors.cover.message}</StyledErrorMessage>
+          )}
+        </StyledLabel> */}
+
```

Comment:
please remove all code instead of commenting it out. If you need it again you can go back to the previous commit and restore it.